tests/test_rooms/test_query_rooms.py

```diff
@@ -78,6 +80,17 @@ def test_calendar_list_function(self, mocked_method):
         get_google_api_calendar_list()
         assert mocked_method.called
 
+    def test_all_remote_rooms_with_argument_returnAll_true(self):
```

Comment:
Please look into your variable naming. Read the conventions -> https://www.python.org/dev/peps/pep-0008/#function-and-variable-names